### PR TITLE
alibuild: init at 1.5.3

### PR DESCRIPTION
**Pull request by ktf**
###### Motivation for this change

Add new package

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/build-managers/alibuild/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, python}:
+
+python.pkgs.buildPythonPackage rec {
```
Comment (ryantm) [inline pkgs/development/tools/build-managers/alibuild/default.nix]: If this is a standalone python application you should use  buildPythonApplication.

File: pkgs/development/tools/build-managers/alibuild/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  doCheck = false;
+  propagatedBuildInputs = [
+    python.pkgs.requests
+    python.pkgs.argparse
+    python.pkgs.pyyaml
+  ];
+
+  patches = [
+    ./fix-argparse.patch
```
Comment (ryantm) [inline pkgs/development/tools/build-managers/alibuild/default.nix]: If this patch is not a nix-specific problem, please report to upstream and add a comment linking to the report.

**Review by ryantm — CHANGES_REQUESTED** [review]
Instead of merging master into your branch, please rebase or add your commits on top of master.

**Comment by ktf** [comment]
I think I've addressed all the comments.

**Pull request merged**